code/modules/food_and_drinks/kitchen_machinery/microwave.dm

```diff
@@ -385,6 +385,12 @@
 			playsound(src, 'sound/items/cig_light.ogg', 50, 1)
 			moveToNullspace()
 
+
+/obj/item/ration_heater/get_temperature()
+	if(!uses)
+		return 0
+	. = ..()
+
```

Comment:
i looked at the attackby and it looks like you would be use this infinity to cauterize is that correct?
it also SHOULD qdel itself after it has no uses left so this shouldn't realllly matter